### ISSUE TITLE
sourcing STM32F042K6T6

### ISSUE THREAD

**Issue by thijsdeschildre** [dekuNukem/exixe#21]
Are there any requirements for the type of STM32F being used?
Availability of these chips is still bad. From what I can see, it needs to support the SPI bus and be in 32-LQFP footprint?

**Comment by dekuNukem** [comment]
Yeah the availability of those chips is still very low, and cost has gone way up, that's why I'm not doing a production run right now.

Anyway, it has to support SPI, in LQFP32, and also has hardware timer PWM output on all cathode pins for smooth dimming. So I STM32F042K6T6 / K4T6 are pretty much the only option. You can also try QFN32 variants, but that would require redesigning the circuit board a bit.
